npm/ng-packs/packages/theme-shared/extensions/src/lib/models/entity-props.ts

```diff
@@ -31,6 +31,7 @@ export class EntityProp<R = any> extends Prop<R> {
   readonly component?: Type<any>;
   readonly enumList?: Array<ABP.Option<any>>;
   readonly tooltip?: string;
+  readonly columnVisible: ColumnPredicate;
```

Comment:
options.visible, already exists. what is the different ? I think, we can use that.